topics/ui/controls/combo_box.md

```diff
@@ -23,79 +23,86 @@ comboBox.setEditable(true);
 
 ## When to use
 
-Use a combo box if:
+Use a combo-box if:
 
-* The user needs to select a value or enter a custom value.
+* The user needs to select a value or enter a custom value:
```

Comment:
Please rewrite each rule as a separate section, see "Split button as an example". Here we could have two rules: "Select value or enter custom", "Return to previous values"

Reply:
Done. Could you please check the headings?